app/assets/stylesheets/publications-table.scss

```diff
@@ -80,3 +80,9 @@
     }
   }
 }
+
+.govuk-pagination {
+  .govuk-pagination__item.govuk-pagination__item--current {
+    color: #ffffff;
+  }
+}
```

Comment:
We should only really need the `.govuk-pagination__item--current` class name here: given the naming system it doesn't need to be nested in or extending any other class. 

Also we should use the `govuk-colour("white")` variable to set the colour.

Reply:
Think I'd literally just use 
```
.govuk-pagination__item--current {
  color: govuk-colour("white");
}
```
Should be specific enough since `--current` is extending `govuk-pagination__item`

Reply:
Good point, I've made that change now.